test/lib/pipeline.test.js

```diff
@@ -943,6 +963,24 @@ describe('Pipeline Model', () => {
         });
     });
 
+    describe('get scmInfo', () => {
+        beforeEach(() => {
+            scmMock.lookupScmUri.resolves({
```

Comment:
Can we have a test for the failure case e.g. when lookupScmUri rejects with err? In that case will the getter also throw err?

Reply:
In that case, the getter should also throw err. I added the test case for the failure case when lookupScmUri rejects with err.